Jenkinsfile

```diff
@@ -35,6 +36,24 @@ pipeline {
 			}
 		}
 
+		stage('clear deployment after subsystem test') {
+			steps {
+				sh 'make clear-deployment'
+			}
+		}
+
+		stage('test subsystem with podman') {
```

Comment:
@oshercc is there a way to run this onprem part on a different slave? they are going to run the subsystem that takes around 15 minutes so if we don't run it in parallel we double the runtime for each PR